Tests/FoundationTests/FileManagerExtensionsTests.swift

```diff
@@ -80,6 +80,19 @@ final class FileManagerExtensionsTests: XCTestCase {
         } catch {}
     }
 
+    func testEncode() {
+        let point = CGPoint(x: 1, y: 2) //CGPoint conforms to Encodable
+        let tmpDirPath = NSTemporaryDirectory()
```

Comment:
See [here](https://nshipster.com/temporary-files/). Interesting update regarding how to correctly use temporary directories.